test/wallet-http-test.js

```diff
@@ -883,6 +883,18 @@ describe('Wallet HTTP', function() {
     assert.deepEqual(state, res);
   });
 
+  it('should get all wallet names', async () => {
+    const names = await wallet.client.get(`/wallet/${wallet.id}/name`);
+
+    assert.equal(names.length, 9);
+  });
+
+  it('should only get wallet-owned name resources', async () => {
```

Comment:
It's not actually getting the resource (the on chain DNS data) but more like the "info" (height, value, utxo owner, etc). I think you can just match this phrasing to the last test and say `"should only get wallet-owned names"`